tests/test_qa.py

```diff
@@ -86,9 +89,31 @@ def test_qa_train_effectiveness_bert():
 
     assert after_loss < before_loss
 
+
 def test_qa_test_bert():
     happy_qa = HappyQuestionAnswering("BERT", "mrm8488/bert-tiny-5-finetuned-squadv2")
     result = happy_qa.test("../data/qa/test.csv")
     answer = [QuestionAnsweringResult(answer='October 31st', score=0.9352769255638123, start=0, end=12),
               QuestionAnsweringResult(answer='November 23rd', score=0.9180678129196167, start=12, end=25)]
     assert result == answer
+
+
+def test_qa_train_effectiveness_roberta():
+    """
+    Ensures that HappyQuestionAnswering.train() results in
+    lowering the loss as determined by HappyQuestionAnswering.eval()
+    """
+
+    happy_qa = HappyQuestionAnswering("ROBERTA", "roberta-base")
+    before_loss = happy_qa.eval("../data/qa/train-eval.csv").eval_loss
+    happy_qa.train("../data/qa/train-eval.csv")
+    after_loss = happy_qa.eval("../data/qa/train-eval.csv").eval_loss
+    assert after_loss < before_loss
+
+
+def test_qa_test_roberta():
+    happy_qa = HappyQuestionAnswering("ROBERTA", "roberta-base")
```

Comment:
Use a fine-tuned roberta QA model. That's why the results don't make sense. 
 "deepset/roberta-base-squad2"

Reply:
Updated to use 'deepset/roberta-base-squad2' as opposed to 'roberta-base'